sbc_compassion/models/import_letter_line.py

```diff
@@ -79,10 +81,25 @@ def _set_sponsorship_id(self):
                     ('child_id.code', '=', line.child_code),
                     ('partner_codega', '=', line.partner_codega),
                     ('is_active', '=', True)], order='end_date desc', limit=1)
+                if len(line.sponsorship_id) == 0:
+                    line.sponsorship_id = line.env[
+                        'recurring.contract'].search(
+                            [('child_id.code', '=', line.child_code),
+                             ('partner_codega', '=', line.partner_codega)],
+                            order='end_date desc', limit=1)
```

Comment:
Active sponsorships won't have end_dates so they may be put at the end of the list. It is safer to use `start_date` desc